Puppetfile

```diff
@@ -90,6 +90,7 @@ github "karabiner",      "1.0.6"
 github "launchbar",      "1.2.0", :repo => "dieterdemeyer/puppet-launchbar"
 github "libreoffice",    "4.2.1", :repo => "alphagov/puppet-libreoffice"
 github "macvim",         "1.0.0"
+github "modern_space_cadet", "1.0.7"
```

Comment:
I thought about that, then noticed `notational_velocity` and `emacs_keybindings` were not aligned, so decided to go with it. Happy to realign if people think that is best though.